apps/test/unit/templates/teacherDashboard/AddSectionDialogTest.jsx

```diff
@@ -113,12 +113,12 @@ describe('AddSectionDialog', () => {
       navigateToHrefSpy.restore();
     });
 
-    /*
     it('redirects to new section setup when selecting non-student participant type', () => {
       const newSection = _.cloneDeep(defaultProps.section);
       const wrapper = shallow(
         <AddSectionDialog
           {...defaultProps}
+          userId={90}
```

Comment:
magic number alert - I would add a comment above that "anything evenly divisible by 10 would pass for this test" or "userID % 10 === 0 is in the new section flow"